src/ai/backend/client/cli/admin/scaling_group.py

```diff
@@ -142,17 +153,31 @@ def update(name, description, inactive,
                 scheduler_opts=scheduler_opts,
             )
         except Exception as e:
-            print_error(e)
+            ctx.output.print_mutation_error(
+                e,
+                item_name='scaling_group',
+                action_name='update',
+            )
             sys.exit(1)
         if not data['ok']:
-            print_fail('Scaling group update has failed: {0}'.format(data['msg']))
+            ctx.output.print_mutation_error(
+                msg=data['msg'],
```

Comment:
Currently, the manager-side error message generation routine just quotes the mutation class name, resulting in messages like "no matching modifyscalinggroup".

Could you make an idea to improve it *in the manager side*? (e.g., "no matching scaling group")
Let's make a separate PR to the manager for this.

Reply:
it would be great to inform the reason of the error in the manager side.
this is an another issue from this pr?